### PR TITLE
fix(build): Only build gh-pages if PR is opened against mozilla/fxa

### LINKED ISSUE #811: Make the docs build step a NOP on forked builds
We don't share build secrets with forks, so the `docs` step in Circle fails there. We should see if there is a handy `CIRCLE_` environment variable that lets us distinguish those and turn them into a NOP instead of a build error.

Ref: https://github.com/mozilla/fxa/pull/775#issuecomment-482274797

### PR DESCRIPTION
**Pull request by shane-tomlinson**
fixes #811

Built against a forked repo

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@philbooth - r?

**Comment by shane-tomlinson** [comment]
@mozilla/fxa-devs - ? Compare this with #830 which is against the mozilla repo.

**Comment by shane-tomlinson** [comment]
@vbudhram, @dannycoates - r?

File: .circleci/config.yml [hunk 1]
```diff
@@ -258,7 +258,12 @@ jobs:
           fingerprints:
             - "08:fc:2b:fb:06:0e:8f:0f:01:1f:28:86:83:89:11:28"
       - run: |
-          if [ "$CIRCLE_REPOSITORY_URL" = "git@github.com:mozilla/fxa.git" ]; then
+          # From https://circleci.com/docs/2.0/env-vars/
```
Comment (vbudhram) [inline .circleci/config.yml]: nit: Could use `https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables` to link directly section

**Pull request merged**